charts/opensearch-cluster/templates/opensearch-cluster-cr.yaml

```diff
@@ -0,0 +1,275 @@
+{{- $cluster_name := .Release.Name }}
```

Comment:
In the entire file please indent the template controls (if, end, ...) to have the same indentation as the fields they control. Makes it easier to read.

E.g.:

```yaml
spec:
  {{- if .Values.OpenSearchClusterSpec.bootstrap }}
  bootstrap:
    ...
  {{- end }}
```

Reply:
Done